integrations/slack/src/router.ts

```diff
@@ -23,6 +32,19 @@ export const handleFetchEvent: FetchEventCallback = async (request, context) =>
         ).pathname,
     });
 
+    const encodedScopes = encodeURIComponent(
+        [
+            'chat:write',
+            'channels:join',
+            'channels:read',
+            'channels:history',
```

Comment:
this I introduced for the curation part
```suggestion
```

Reply:
I wonder, since we are potentially getting people to re-install the Slack app at this moment, should we prime it with the permissions needed so that the other feature will immediately work when we push it?